astarte-device-sdk/AstarteDeviceSDK.h

```diff
@@ -62,6 +62,10 @@ class AstarteDeviceSDK : public Hemera::AsyncInitObject
 
     bool sendData(const QByteArray &interface, const QVariantHash &value, const QVariantHash &metadata);
 
+    template <typename T>
+    bool sendData(const QByteArray &interface, const QByteArray &path, const QList<T> &value,
+            const QDateTime &timestamp = QDateTime(), const QVariantHash &metadata = QVariantHash());
+
```

Comment:
let's try to indent this in a more readable way.
I would leave either `template <...>` in the same line of `sendData`, or indent the following line.